pymc/sampling/parallel.py

```diff
@@ -96,13 +102,12 @@ def __init__(
         shared_point,
         draws: int,
         tune: int,
-        rng: np.random.Generator,
-        seed_seq: np.random.SeedSequence,
+        rng_state: RandomGeneratorState,
         blas_cores,
     ):
         # For some strange reason, spawn multiprocessing doesn't copy the rng
         # seed sequence, so we have to rebuild it from scratch
```

Comment:
Comment can be removed/updated? It's the bug you found for numpy<2.0?

Reply:
Yes, you’re right. I’ll update the comment